### DISCUSSION TITLE
Unable to start AeroSpace

### DISCUSSION THREAD

**Discussion by Slijeff** [nikitabobko/AeroSpace#735] [questions-and-answers] [answered]
Hi, I tried to open Aerospace by clicking the icon, go to the application folder and right click open, and also using Spotlight search Aerospace then enter, but no respond after performing those actions. Aerospace icon is not shown in the menu bar. It might have something to do with macos 15.1.1 which I updated yesterday, but I'm not sure.

Below are some info by running `aerospace -v`:
```
aerospace CLI client version: 0.15.2-Beta b6cf82771f245ab7349a93baf8709e171537ff58
AeroSpace.app server version: Unknown. The server is not running
```

System info:
```
OS: macOS Sequoia 15.1.1 arm64
Host: MacBook Pro (14-inch, 2021)
Kernel: Darwin 24.1.0
Uptime: 2 hours, 5 mins
Packages: 626 (nix-system), 45 (nix-default), 2 (brew), 16 (brew-cask)
Shell: zsh 5.9
Display (Color LCD): 3024x1964 @ 120 Hz (as 1512x982) in 14" [Built-in]
DE: Aqua
WM: Quartz Compositor
WM Theme: Multicolor (Light)
Font: .AppleSystemUIFont [System], Helvetica [User]
Cursor: Fill - Black, Outline - White (32px)
Terminal: WezTerm 20240203-110809-5046fc22
Terminal Font: JetBrains Mono
CPU: Apple M1 Pro (8) @ 3.23 GHz
GPU: Apple M1 Pro (14) @ 1.30 GHz [Integrated]
Memory: 10.01 GiB / 16.00 GiB (63%)
Swap: Disabled
Disk (/): 136.86 GiB / 460.43 GiB (30%) - apfs [Read-only]
Local IP (en0): 10.194.206.54/14
Battery (bq40z651): 82% (9 hours, 50 mins remaining) [Discharging]
Locale: en_US.UTF-8
```

**Comment by gldtn** [comment]
> Below are some info by running `aerospace -v`:
> 
> ```
> aerospace CLI client version: 0.15.2-Beta b6cf82771f245ab7349a93baf8709e171537ff58
> AeroSpace.app server version: Unknown. The server is not running
> ```

If your getting a response from that, aerospace should be running in the background, it does not show an icon in the menu bar. Instead you should have the active space number 1|2 or something similar. If you click on it, you get a menu.. Are the window tiling at all? I'm running Sequoia 15.2 beta and Aerospace is working fine on my end.

**Reply by Slijeff** [reply]
I don't see the active space number, and the windows are not tiling at all. Pressing the keybindings have no effect.

**Comment by Slijeff** [comment, marked as answer]
Okay I found the issue. There's something wrong with my aerospace config, so AeroSpace aborts on start. But there's no error message whatsoever. After I switched to the default config then it's able to start.

**Reply by gldtn** [reply]
Usually error popups up as should as you try reloading, but I have had a couple cases where it reloaded without errors. Glad you were able to figure it out.
